{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/config/jinja2.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+import random
+from math import ceil, floor
+
+from django.conf import settings
+
+
+def random_chart(
```

Comment:
I think we'd do well to have a one-liner about why this code is here, even if it's just `# Sample code from the template`

Reply:
yeah agreed